### PR TITLE
ref tasks #9726 user_mangement

### PR DESCRIPTION
**Pull request by nguyenthithu1612**
https://edu-redmine.sun-asterisk.vn/issues/9726

### CODE REVIEW

**Comment by nguyenthithu1612** [comment]
usermanage

**Comment by nguyenthithu1612** [comment]
ready

File: app/Http/Controllers/Admin/UserController.php [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        // User::updateOrCreate(['id' => $request->user_id],
+
+        //         ['username' => $request->name, 'detail' => $request->detail]);
```
Comment (sonnguyen8x) [inline app/Http/Controllers/Admin/UserController.php]: không dùng thì xoá đi

File: app/Http/Controllers/Admin/UserController.php [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+     */
+    public function index(Request $request)
+    {
+        if ($request->ajax()) {
+            $data = User::latest()->get();
+
+            return Datatables::of($data)
+                    ->addIndexColumn()
+                    ->addColumn('action', function($row){
+                        $btn = '<a href="javascript:void(0)" data-toggle="tooltip"  data-id="'.$row->id.'" data-original-title="Edit" class="edit btn btn-primary btn-sm editProduct">' . __('label.edit') . '</a>';
```
Comment (sonnguyen8x) [inline app/Http/Controllers/Admin/UserController.php]: convention

**Pull request merged**